.github/workflows/check.yaml

```diff
@@ -30,3 +30,16 @@ jobs:
         run: postgrest-lint
       - name: Run style check (auto-format with `nix-shell --run postgrest-style`)
         run: postgrest-style-check
+
+  commit-check:
+    name: Commit Message Check
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Setup Nix Environment
+        uses: ./.github/actions/setup-nix
+        with:
+          authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+          tools: commit.check.bin
```

Comment:
I guess, we need to update the `CACHIX_AUTH_TOKEN` to support this new workflow.

Reply:
The CI addition of this is failing with `error: attribute 'commit' in selection path 'commit.check.bin' not found`. So, I made a wild guess that it might related to it, but I am not sure.